mayastor/src/bdev/nexus/nexus_state.rs

```diff
@@ -0,0 +1,36 @@
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+#[derive(Debug, Deserialize, PartialEq, Default, Serialize, Clone)]
+pub struct NexusConfigVersion1 {
+    pub name: String,
+    pub labels: Vec<String>,
```

Comment:
whats "labels" ?

Reply:
from the tests it seems that labels should be "key?

Reply:
The NexusConfigVersion<N> structs are really just placeholders for whatever we finally decide to define as our "data structure". The fields are just made up nonsense. The point was to demonstrate that we could handle different Versions of the struct" by wrapping them in an enum. You are also right in that we should have just a "version1" for now and try not to add newer versions too often. But the mechanism for handling multiple versions should be there right from the start.